maestro-client/src/main/java/maestro/drivers/AndroidDriver.kt

```diff
@@ -152,6 +152,10 @@ class AndroidDriver(
         shell("pm clear $appId")
     }
 
+    override fun clearKeychain() {
```

Comment:
Users may expect this to be cleared on Android: https://developer.android.com/reference/android/security/KeyChain